cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -427,107 +509,144 @@ func transcribeFiles(workerID int, wg *sync.WaitGroup, client *cubic.Client,
 		if outputFormat == "timeline" {
 			cfg.EnableWordConfidence = true
 			cfg.EnableWordTimeOffsets = true
-
 		}
 
+		// create buffer for file's results
+		var fileResponses []responseResults
```

Comment:
Each section returns a slice of `[]*RecognitionResult`.  I suppose it's fine to collapse those into a single slice instead of multiple slices.  
This grew organically from passing off the array as soon as we got it to the results, but I think it's appropriate to change.